### ISSUE TITLE
Uncaught TypeError: Cannot read property 'querySelector' of null

### ISSUE THREAD

**Issue by tjhorner** [thomaslindstrom/color-picker#63]
1. Open color picker.
2. Color picker is misplaced and error is thrown.

**Atom Version**: 0.165.0
**System**: linux 3.13.0-32-generic
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.4.4

### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of null

```
At /home/tj/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101

TypeError: Cannot read property 'querySelector' of null
  at ColorPickerView.module.exports.ColorPickerView.open (/home/tj/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101:66)
  at Object.module.exports.open (/home/tj/.atom/packages/color-picker/lib/ColorPicker.coffee:108:23)
  at atom-text-editor.atom.commands.add.color-picker:open (/home/tj/.atom/packages/color-picker/lib/ColorPicker.coffee:20:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:167:20)

```

### Commands

```
     -1:01.2 command-palette:toggle (input.hidden-input)
     -1:00.1 core:confirm (input.hidden-input)
     -1:00.1 settings-view:open (input.hidden-input)
     -0:54.7 core:confirm (input.hidden-input)
  4x -0:45.4 color-picker:open (input.hidden-input)
     -0:03.1 editor:newline (input.hidden-input)
     -0:01.4 core:backspace (input.hidden-input)
     -0:00.0 color-picker:open (input.hidden-input)
```

### Config

```
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-jade, v0.1.0
color-picker, v1.4.4

# Dev
No dev packages
```

/cc @atom/core

**Comment by thomaslindstrom** [comment]
Hi! Please update your Atom version and try again.

**Comment by jagged3dge** [comment]
Still facing the error.

**Atom Version**: 0.204.0
**System**: Microsoft Windows 7 Ultimate 
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v2.0.2


### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of null

```
At C:\Users\Gary\.atom\packages\color-picker\lib\ColorPicker-view.coffee:410

TypeError: Cannot read property 'querySelector' of null
  at Object.open (C:\Users\Gary\.atom\packages\color-picker\lib\ColorPicker-view.coffee:410:49)
  at atom-text-editor.<anonymous> (C:\Users\Gary\.atom\packages\color-picker\lib\ColorPicker.coffee:42:68)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\Gary\AppData\Local\atom\app-0.204.0\resources\app.asar\src\command-registry.js:238:29)
  at C:\Users\Gary\AppData\Local\atom\app-0.204.0\resources\app.asar\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\Gary\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\Gary\AppData\Local\atom\app-0.204.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (C:\Users\Gary\AppData\Local\atom\app-0.204.0\resources\app.asar\src\window-event-handler.js:178:20)

```

### Commands

```
     -0:20.9.0 editor:copy-path (input.hidden-input)
     -0:18.9.0 color-picker:open (input.hidden-input)
     -0:04.4.0 editor:consolidate-selections (input.hidden-input)
     -0:04.4.0 core:cancel (input.hidden-input)
     -0:02.2.0 color-picker:open (input.hidden-input)
```

### Config

```json
{
  "core": {
    "themes": [
      "neutron-ui",
      "flatland-dark"
    ],
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "linter-eslint",
      "linter-scss-lint",
      "linter"
    ],
    "autoHideMenuBar": true
  }
}
```

### Installed Packages

```coffee
# User
atom-alignment, v0.12.1
atom-color-highlight, v3.0.9
color-picker, v2.0.2
docblockr, v0.7.2
emmet, v2.3.10
flatland-dark, v0.1.6
git-log, v0.4.1
minimap, v4.9.0
neutron-ui, v0.4.0

# Dev
No dev packages
```

**Comment by Harat** [comment]
I have the same error on Mac Yosemite 10.10.3 - Atom Version 0.209.0 (0.209.0)

[[ Uncaught TypeError: Cannot read property 'querySelector' of null ]]

**Comment by thomaslindstrom** [comment]
Hi! This should be fixed in `v2.0.6`. Please also activate `Shadow DOM` in the Atom Settings for an improved experience!
